Signal/src/Models/OWSCall.h

```diff
@@ -15,9 +15,9 @@ typedef enum : NSUInteger {
     kCallMissed = 3,
     kCallOutgoingIncomplete = 4,
     kCallIncomingIncomplete = 5,
-    // kGroupUpdateJoin has been deprecated.
-    kGroupUpdateLeft = 7,
-    kGroupUpdate = 8,
+    // kGroupUpdateJoin = 6, has been deprecated.
+    // kGroupUpdateLeft = 7, has been deprecated.
+    // kGroupUpdate = 8, has been deprecated.
```

Comment:
My inclination is to _not_ comment them out but to rename them instead to something like kGroupUpdateJoin_OBSOLETE.  We want to make sure that they're handled properly, and by leaving these values in the enum we'll get valuable compile-time warnings about incomplete switch statements.